dora/src/bytecode/generator.rs

```diff
@@ -260,9 +264,11 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
             let end_lbl = self.gen.create_label();
 
             let cond_reg = self.visit_expr(&expr.cond, DataDest::Alloc);
+            self.gen.set_position(expr.pos);
```

Comment:
Do we really need to call set_position for jump instructions?

Reply:
More or less same as above, but if we don't emit a position for every bytecode, but only for the relevant ones.